spk/nzbget/Makefile

```diff
@@ -49,9 +49,7 @@ nzbget_extra_install: $(STAGING_DIR)/share/nzbget/scripts/nzbToMedia $(STAGING_D
 
 $(STAGING_DIR)/share/nzbget/scripts/nzbToMedia:
 	cd $(STAGING_DIR)/share/nzbget/scripts && git clone https://github.com/clinton-hall/nzbToMedia.git
-	rm -fr $(STAGING_DIR)/share/nzbget/scripts/nzbToMedia/.git*
```

Comment:
I confirmed that this works:

INFO	Mon Jun 01 2015 23:06:17	nzbToNzbDrone: [23:06:17] [INFO]::MAIN: No update needed
INFO	Mon Jun 01 2015 23:06:17	nzbToNzbDrone: [23:06:17] [INFO]::MAIN: Checking if git needs an update
INFO	Mon Jun 01 2015 23:06:17	nzbToNzbDrone: [23:06:17] [INFO]::MAIN: Checking database structure...
INFO	Mon Jun 01 2015 23:06:15	nzbToNzbDrone: [23:06:15] [INFO]::MAIN: Restarting nzbToMedia with ['/usr/local/python/bin/python2', '/usr/local/nzbget/share/nzbget/scripts/nzbToMedia/nzbToNzbDrone.py']
INFO	Mon Jun 01 2015 23:06:12	nzbToNzbDrone: [23:06:12] [INFO]::MAIN: Auto-Updating nzbToMedia, Please wait ...
INFO	Mon Jun 01 2015 23:06:12	nzbToNzbDrone: [23:06:12] [INFO]::MAIN: There is a newer version available (you're 1 commit behind)
INFO	Mon Jun 01 2015 23:06:11	nzbToNzbDrone: [23:06:11] [INFO]::MAIN: Checking if git needs an update